packages/measured-reporting/lib/registries/DimensionAwareMetricsRegistry.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Simple registry that stores Metrics by name and dimension
+ * @private
+ */
+class DimensionAwareMetricsRegistry {
```

Comment:
Somehow the docs for this 404s when clicking through from [Reporter.setRegistry](https://yaorg.github.io/node-measured/Reporter.html#setRegistry__anchor) docs: https://yaorg.github.io/node-measured/DimensionAwareMetricsRegistry.html

Reply:
Good catch, that class has a @private so its docs didn't get generated.

- [ ] Make DimensionAwareMetricsRegistry public, since it can be overridden.